k8_cortx_cloud/solution.yaml

```diff
@@ -24,11 +24,9 @@ solution:
     rancher: ghcr.io/seagate/local-path-provisioner:v0.0.20
     busybox: ghcr.io/seagate/busybox:latest
   common:
-    setup_size: large
```

Comment:
why we are removing setup_size from solution.yaml and not from _config.tpl

Reply:
Removed from both the places